lib/google_crawler/search/upload.ex

```diff
@@ -0,0 +1,27 @@
+defmodule GoogleCrawler.Search.Upload do
+  alias Ecto.Multi
+  alias GoogleCrawler.Accounts.User
+  alias GoogleCrawler.Repo
+  alias GoogleCrawler.Search.Keyword
+
+  def process(%User{} = user, %Plug.Upload{} = upload) do
+    upload.path
+    |> keyword_values()
+    |> Enum.reduce(Multi.new(), fn name, multi ->
+      Multi.insert(
+        multi,
+        {:keyword, name},
+        Keyword.changeset(%Keyword{}, %{name: String.trim(name), user_id: user.id})
+      )
+    end)
+    |> Repo.transaction()
+  end
+
+  defp keyword_values(upload_path) do
+    upload_path
+    |> File.stream!()
+    |> CSV.decode!(seperator: ",")
+    |> Enum.take(100)
```

Comment:
This will limit the keyword per file to a hundred, feels like this one is configuration value. Should we move `100` to the module attribute or env?

Reply:
Fixed in https://github.com/ankitkalia1195/elixir-google-crawler/pull/34/commits/f300517565b4673169c7ca8270285d66e2c1cbfe